datasets/janelia-cosem.yaml

```diff
@@ -0,0 +1,25 @@
+Name: COSEM on AWS
+Description: High resolution images of subcellular structures.
+Documentation: https://docs.opendata.aws/cosem/readme.md
+Contact: COSEMData@janelia.hhmi.org
+ManagedBy: "[COSEM@HHMI/Janelia](https://www.janelia.org/project-team/cosem)"
+UpdateFrequency: New datasets and derived data are added as soon as they are available.
+Tags:
+  - cell biology
+  - organelle
+  - electron microscopy
+  - computer vision
+  - machine learning
+License: CC-BY-4.0
+Resources:
+  - Description: Raw FIB-SEM datasets and derived data.
+    ARN: arn:aws:s3:::janelia-cosem
+    Region: us-east-1
+    Type: S3 Bucket
+  Publications:
```

Comment:
Missing schema!

DataAtWork
  Tools & Applications:
  Tutorials:
  Publications:
    - Title: Enhanced FIB-SEM systems for large-volume 3D imaging  
      URL: https://elifesciences.org/articles/25916
      AuthorName: C. Shan Xu, Kenneth J. Hayworth, Zhiyuan Lu, Patricia Grob, Ahmed M. Hassan, José G. García Cerdán, Krishna K. Niyogi, Eva Nogales, Richard J. Weinberg, Harald F. Hess.      
    - Title: Correlative three-dimensional super-resolution and block-face electron microscopy of whole vitreously frozen cells.
      URL: https://science.sciencemag.org/content/367/6475/eaaz5357
      AuthorName: David P. Hoffman1, Gleb Shtengel, C. Shan Xu, Kirby R. Campbell, Melanie Freeman, Lei Wang, Daniel E. Milkie, H. Amalia Pasolli, Nirmala Iyer, John A. Bogovic, Daniel R. Stabley, Abbas Shirinifard, Song Pang, David Peale, Kathy Schaefer, Wim Pomp, Chi-Lun Chang, Jennifer Lippincott-Schwartz, Tom Kirchhausen1, David J. Solecki, Eric Betzig, Harald F. Hess.

Reply:
Sorry about that, it should be fixed now. We don't have tutorials or out data portal site up yet, but I'll submit a follow-up PR when those are ready.

Reply:
No worries! Can you do me a favor and merge upstream changes? I added the cell biology tag to tags.yaml so that should be taken care of.

Reply:
done!